sqs/src/main/scala/akka/stream/alpakka/sqs/scaladsl/SqsAckFlow.scala

```diff
@@ -30,7 +31,7 @@ object SqsAckFlow {
    */
   def apply(queueUrl: String, settings: SqsAckSettings = SqsAckSettings.Defaults)(
       implicit sqsClient: SqsAsyncClient
-  ): Flow[MessageAction, SqsAckResult[SqsResponse], NotUsed] =
+  ): Flow[MessageAction, SqsAckResult[SdkPojo], NotUsed] =
```

Comment:
This improves the consistency for Batch actions.
In this case we want to give the metadata entry associated to the single `MessageAction` and not the response which was sent back from the whole batch.

As mentioned in #625, in the batch `SqsResponse`, we don't have any way to identify the specific metadata entry for the `MessageAction` because we don't know the batch id. (we may be able to do it by comparing md5 hashes of the messages but that's not nice)

Reply:
But here we don't have batches. We can use `SdkPojo` for the batch actions.